azurelinuxagent/distro/default/rdma.py

```diff
@@ -0,0 +1,74 @@
+# Windows Azure Linux Agent
+#
+# Copyright 2016 Microsoft Corporation, SUSE LLC, Robert Schweikert
```

Comment:
On 06/03/2016 06:28 PM, Ahmet Alp Balkan wrote:
>> @@ -0,0 +1,74 @@
>> +# Windows Azure Linux Agent
>> +#
>> +# Copyright 2016 Microsoft Corporation, SUSE LLC, Robert Schweikert
> 

All done, ready for merge


-- 
Robert Schweikert                   MAY THE SOURCE BE WITH YOU
Public Cloud Architect                         LINUX
rjschwei@suse.com
IRC: robjo